docs/_assets/theme.css

```diff
@@ -45,6 +48,9 @@ html {
     --rocket-search-highlight-color: #41ffb0;
     --rocket-search-hover-background-color: #6b717a;
     --rocket-search-fill-color: #fff;
+    --primary-text-inverse-color: #2c3e50;
```

Comment:
@gerjanvangeest  Can you add you theme changes to `@rocket/launch` as well, and bump that package?